bin/oneoff/backfill_data/user_facilitator_bios.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This script migrates facilitator bios from Pegasus markdown files to the dashboard database
+
+require 'redcarpet'
+require 'redcarpet/render_strip'
+require 'ruby-progressbar'
+
+require_relative '../../../dashboard/config/environment'
+
+facilitator_bio_paths = Dir[pegasus_dir('sites.v3/code.org/views/workshop_affiliates/*_bio.md')]
+progress_bar = ProgressBar.create(total: facilitator_bio_paths.size, format: 'Migrated[%c/%C]: |%W| %a')
+markdown_to_text_converter = Redcarpet::Markdown.new(Redcarpet::Render::StripDown)
+
+facilitator_bio_paths.each do |facilitator_bio_path|
+  user_id = File.basename(facilitator_bio_path, '_bio.md').to_i
```

Comment:
Should we check if the user with this id both exists and is a facilitator?

Related, should we add any validation to `FacilitatorInfo` that the user is actually a facilitator?